robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -223,7 +223,8 @@ public static final class SHOOTER_LIMELIGHT {
   }
 
   public static final class SHOOTER_PIVOT {
-    public static final double INTAKE_FROM_SOURCE_ROTATIONS = 0;
+    public static final double INTAKE_FROM_SOURCE_SETPOINT = 0;
```

Comment:
I would use rotation over setpoint. We will be setting the pivot to a value relative to the helium absolute encoder which will be between 0 and 1.

Reply:
The only reason I changed it is because before it sounded like we were going off of motor rotations